mediapipe/calculators/ovms/openvinoinferencecalculator.cc

```diff
@@ -461,6 +461,10 @@ class OpenVINOInferenceCalculator : public CalculatorBase {
         ::InferenceInput input;
         ::InferenceOutput output;
         for (const std::string& tag : cc->Inputs().GetTags()) {
+            if (cc->Inputs().Tag(tag).IsEmpty()) {
+                LOG(INFO) << "OpenVINOInferenceCalculator expects all input packets at the same time for each call to Process().";
+                RET_CHECK(false);
+            }
```

Comment:
So this calculator will work only with configuration when all inputs are available. I guess that's expected because we will not be able to run any inference partial inputs?

Reply:
Correct